pkg/cmd/import.go

```diff
@@ -32,7 +32,7 @@ const projectTemplateFileName = "doppler-template.yaml"
 var importCommand = &cobra.Command{
 	Use:   "import",
 	Short: "Import projects into your Doppler workplace",
-	Args:  cobra.MaximumNArgs(1),
+	Args:  cobra.NoArgs,
```

Comment:
This has a template flag where you can specify what file to use (but I guess this is a flag and not an arg?).

Reply:
Flags are handled a bit differently than args. Ideally this command should probably accept the template path via flag or via arg (like our other commands do- e.g. `doppler projects create`).